### PR TITLE
Add webrtc test via CI

### PR DESCRIPTION
**Pull request by patrickelectric**
The test runs fine locally with:
```
cargo run --verbose -- --mavlink udpin:0.0.0.0:6666 --verbose --enable-thread-counter --enable-webrtc-task-test --default-settings WebRTCTest --reset
```

No idea why is failing on CI, crashing in a wrong scenario. I believe that @joaoantoniocardoso async rework may fix the issue on CI for this patch.

### CODE REVIEW

File: .github/workflows/test_webrtc_leak.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      uses: nanasess/setup-chromedriver@v2
+
+    - name: Install libraries
+      run: |
+        sudo apt install -y --no-install-recommends ca-certificates gnupg
+        sudo apt update
+        sudo apt install -y chromium-chromedriver
+        sudo apt install -y libunwind-dev
+        sudo apt install -y --no-install-recommends libclang-dev libssl-dev pkg-config libmount-dev libsepol-dev libselinux1-dev libglib2.0-dev libgudev-1.0-dev \
+          gstreamer1.0-tools libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev libgstreamer-plugins-good1.0-dev libgstreamer-plugins-bad1.0-dev gstreamer1.0-plugins-ugly libgstrtspserver-1.0-dev
```
Comment (joaoantoniocardoso) [inline .github/workflows/test_webrtc_leak.yml]: maybe:
```suggestion
        sudo apt install -y --no-install-recommends libclang-dev libssl-dev pkg-config libmount-dev libsepol-dev libselinux1-dev libglib2.0-dev libgudev-1.0-dev \
          gstreamer1.0-tools libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev libgstreamer-plugins-good1.0-dev libgstreamer-plugins-bad1.0-dev gstreamer1.0-plugins-ugly libgstrtspserver-1.0-dev libgstreamer1.0-0 libgstreamer-plugins-base1.0-0 libgstreamer-plugins-bad1.0-0 libgstrtspserver-1.0-0
```

Comment (patrickelectric) [reply .github/workflows/test_webrtc_leak.yml]: fixed by gstreamer-1.0-x

File: src/custom/test.rs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+use crate::stream::types::*;
+use crate::video::{self, types::*, video_source_gst::VideoSourceGstType};
+use crate::video_stream::types::VideoAndStreamInformation;
+
+pub fn take_webrtc_stream() -> Vec<VideoAndStreamInformation> {
+    let size = STANDARD_SIZES.last().unwrap();
+    vec![VideoAndStreamInformation {
+        name: format!("WebRTC fake stream for thread leak"),
+        stream_information: StreamInformation {
+            endpoints: vec![Url::parse("udp://0.0.0.0:8554/video_0").unwrap()],
```
Comment (joaoantoniocardoso) [inline src/custom/test.rs:12]:
```suggestion
            endpoints: vec![Url::parse("udp://0.0.0.0:8554/test").unwrap()],
```

Comment (patrickelectric) [reply src/custom/test.rs]: Done!

**Comment by patrickelectric** [comment]
@joaoantoniocardoso I believe that it's ready to merge, indeed CI is failing, but at least we know that there is something wrong.

**Review by joaoantoniocardoso — APPROVED** [review]

**Pull request merged**